bundles/org.openhab.ui/web/src/pages/settings/items/item-edit.vue

```diff
@@ -159,12 +160,10 @@ export default {
       if (this.currentTab === 'code') {
         if (!this.fromYaml()) return Promise.reject()
       }
-      if (!this.item.name) return // user cannot change name
+      if (!this.item.name) return this.$f7.dialog.alert('Please give Item a valid name').open() // user cannot change name
```

Comment:
(not blocking, for later)
```suggestion
      if (!this.item.name) return this.$f7.dialog.alert('Please give the Item a valid name').open() // user cannot change name
```